packages/gatsby/cache-dir/app.js

```diff
@@ -8,9 +8,9 @@ import emitter from "./emitter"
 import { apiRunner, apiRunnerAsync } from "./api-runner-browser"
 import { setLoader, publicLoader } from "./loader"
 import DevLoader from "./dev-loader"
-import syncRequires from "./sync-requires"
+import syncRequires from "sync-requires"
```

Comment:
Naming should be different, this can clash with other node-module names. Maybe make it part of gatsby package 🤷 
```suggestion
import syncRequires from "gatsby/virtual/sync-requires.js"
```

Reply:
similar things I considered was using something like `@virtual/sync-requires` or `$virtual/sync-requires` (can throw in `gatsby` keyword there as well). I think using some special character like that there would be good idea for "greppability" of code (searching for `@virtual` in code base to see usage)

Reply:
I would go for `$virtual` cause `@virtual` could possibly be an npm org

Reply:
Updated to use ~`$virtual/sync-requires`